backend/src/v5/middleware/permissions/components/teamspaces.js

```diff
@@ -57,4 +58,17 @@ TeamspacePerms.isTeamspaceMember = async (req, res, next) => {
 	}
 };
 
+TeamspacePerms.isModuleEnabled = (moduleName) => async (req, res, next) => {
+	try {
+		const addOns = await getAddOns(req.params.teamspace);
+		if (addOns.modules && addOns.modules.includes(moduleName)) {
+			await next();
+		} else {
+			respond(req, res, createResponseCode(templates.notAuthorized, `${moduleName} module is not enabled in this teamspace`));
```

Comment:
can we make a new template for this?

It's technically not a notAuthorized situation thus it shouldn't use that code